src/test/java/RenameAPlaylist.java

```diff
@@ -1,53 +1,66 @@
+    String  newPlaylistName= "Best";
 
     @Test
-    public void renameSelectedPlaylist(){
+
+    public void renameSelectedPlaylist() {
         login();
         chooseAPlaylist();
+        nameNewPlaylist();
+        Assert.assertEquals(newPlaylistName, getTextNameNewPlaylist());
 
-wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(css)));
-        WebElement renamedPlaylist=driver.findElement(By.cssSelector(css));
-        renamedPlaylist.sendKeys(newPlaylistName+"\n");
-        Assert.assertTrue(renamedPlaylist.isDisplayed());
     }
+    public void login() {
+        WebElement emailField = driver.findElement(By.xpath("//input[@placeholder='Email Address']"));
+        waitForElementToBeClickable(emailField).click();
+        emailField.sendKeys(email);
+
+        WebElement passwordField = driver.findElement(By.xpath("//input[@type='password']"));
+        waitForElementToBeClickable(passwordField).click();
+        passwordField.sendKeys(Password);
+
+        WebElement submitButton =driver.findElement(By.xpath("//button[@type='submit']"));
+        waitForVisibilityOfElement(submitButton).click();
 
+
+    }
     private void chooseAPlaylist() {
-        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(xpath)));
-        WebElement playlist = driver.findElement(By.xpath(xpath));
+        WebElement playlist = driver.findElement(newPlaylistLocator);
+        waitForVisibilityOfElement(playlist);
+        Actions actions=new Actions(driver);
         actions.doubleClick(playlist).perform();
     }
+    private void nameNewPlaylist() {
+        WebElement renamedPlaylist=driver.findElement(getNewPlaylistTextFieldLocator);
+        renamedPlaylist.sendKeys((Keys.chord(Keys.CONTROL + "a", Keys.BACK_SPACE)));
+        waitForVisibilityOfElement(renamedPlaylist);
+        renamedPlaylist.sendKeys(newPlaylistName);
+        renamedPlaylist.sendKeys(Keys.ENTER);
+    }
 
-    private void login() {
-        By emailLocator = By.xpath("//input[@placeholder='Email Address']");
-        wait.until(ExpectedConditions.elementToBeClickable(emailLocator));
-        WebElement emailField = driver.findElement(emailLocator);
-        emailField.click();
-        emailField.sendKeys(email);
 
-        By passwordLocator = By.xpath("//input[@type='password']");
-        wait.until(ExpectedConditions.elementToBeClickable(passwordLocator));
-        WebElement passwordField = driver.findElement(passwordLocator);
-        passwordField.click();
-        passwordField.sendKeys(Password);
 
-        By submitButtonLocator = By.xpath("//button[@type='submit']");
-        wait.until(ExpectedConditions.elementToBeClickable(submitButtonLocator));
-        driver.findElement(submitButtonLocator).click();
 
-    }
+        private String getTextNameNewPlaylist() {
+            return driver.findElement(newPlaylistLocator).getText();
+
+        }
 
 
     }
 
+
+
```

Comment:
**Please delete the empty line!** «An Empty Line is a "Code Smell» 

The short answer is,They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce.

Use "reformat code in Intellij"

Once the rules are configured, you can invoke the Reformat code action that changes the formatting of your code in the selected code fragment, within a file, or across your entire project.

![image](https://user-images.githubusercontent.com/104761482/200171020-17b32367-0631-4f46-b776-bb4053315724.png)